### ISSUE TITLE
Question about the parameter

### ISSUE THREAD

**Issue by Heycoli** [facebookresearch/DeepHandMesh#5]
in the file model.py, there are two variables, target_joint and mesh_refined_v.detach() at line 124 and 126. What are they? I don't seem to be able to find where they are defined

**Comment by mks0601** [comment]
Sorry, I edited the codes.
